### PR TITLE
Feature: Chirped beams

### PR DESCRIPTION
**Pull request by joeydumont**
Add a feature in all drivers to support a spectral phase of the form exp(i*phi(omega)), where phi(omega) is Taylor expanded around a given frequency omega_0.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -185,6 +185,18 @@ target_link_libraries("IntegrationStrattoMosaic"  Cubature)
...
+add_executable("IntegrationStrattoMosaicSG" "simulations/IntegrationStrattoMosaicSG.cpp")
+target_link_libraries("IntegrationStrattoMosaicSG" ${LIBS} ${CUBALIBS} ${STRATTOLIBS})
+target_link_libraries("IntegrationStrattoMosaicSG"  Cubature)
+target_link_libraries("IntegrationStrattoMosaicSG"  ${cuba_dir}/libcuba.a)
+install(TARGETS "IntegrationStrattoMosaicSG" RUNTIME DESTINATION bin)
+
+add_executable("IntegrationStrattoMosaicSGZernike" "simulations/IntegrationStrattoMosaicSGZernike.cpp")
+target_link_libraries("IntegrationStrattoMosaicSGZernike" ${ZERNIKELIBS} ${LIBS} ${CUBALIBS} ${STRATTOLIBS})
+target_link_libraries("IntegrationStrattoMosaicSGZernike"  Cubature)
+target_link_libraries("IntegrationStrattoMosaicSGZernike"  ${cuba_dir}/libcuba.a)
```
Comment (filliong) [inline CMakeLists.txt]: I would change ${cuba_dir}/libcuba.a to Cuba, see the version in branch develop.

File: CMakeLists.txt [hunk 2]
```diff
@@ -185,6 +185,18 @@ target_link_libraries("IntegrationStrattoMosaic"  Cubature)
 target_link_libraries("IntegrationStrattoMosaic"  ${cuba_dir}/libcuba.a)
 install(TARGETS "IntegrationStrattoMosaic" RUNTIME DESTINATION bin)
 
+add_executable("IntegrationStrattoMosaicSG" "simulations/IntegrationStrattoMosaicSG.cpp")
+target_link_libraries("IntegrationStrattoMosaicSG" ${LIBS} ${CUBALIBS} ${STRATTOLIBS})
+target_link_libraries("IntegrationStrattoMosaicSG"  Cubature)
+target_link_libraries("IntegrationStrattoMosaicSG"  ${cuba_dir}/libcuba.a)
```
Comment (filliong) [inline CMakeLists.txt]: I would change ${cuba_dir}/libcuba.a to Cuba, see the version in branch develop.

Comment (joeydumont) [reply CMakeLists.txt]: Alright. I'll have to rebase on the new develop branch though.

**Comment by joeydumont** [comment]
@filliong Ready for re-review.

**Pull request merged**